### PR TITLE
helm: Add llm-faqgen-tgi support

### PR DESCRIPTION
**Pull request by lianhao**
## Description

- Add llm-faqgen-tgi support
- Add variant_xxx-values.yaml to support variants of the same chart

## Issues

Related to issue #280.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Comment by lianhao** [comment]
we expect to see test failure of "E2E test with helm charts / variant_faqgen-value" due to bug opea-project/GenAIComps#700. Let's wait until PR opea-project/GenAIComps#701 get merged

File: helm-charts/common/llm-uservice/templates/tests/test-pod.yaml [hunk 1]
```diff
@@ -25,6 +25,12 @@ spec:
               -X POST \
               -d '{"query":"Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5."}' \
               -H 'Content-Type: application/json' && break;
+          {{- else if contains "llm-faqgen-tgi" .Values.image.repository }}
+          # Try with docsum endpoint
```
Comment (yongfengdu) [inline helm-charts/common/llm-uservice/templates/tests/test-pod.yaml:29]: s/docsum/faqgen/

**Review by yongfengdu — CHANGES_REQUESTED** [review]

File: helm-charts/common/llm-uservice/templates/tests/test-pod.yaml (hunk 1 above)
Comment (lianhao) [reply helm-charts/common/llm-uservice/templates/tests/test-pod.yaml]: done

**Review by yongfengdu — APPROVED** [review]

**Review by zhlsunshine — APPROVED** [review]
LGTM, need to integrate to GMC in later.

**Pull request merged**